src/backend/services/checkout-api/src/messagging/publisher.ts

```diff
@@ -32,6 +33,10 @@ export class Publisher {
     const kafka = new Kafka({
       clientId: this.clientId,
       brokers: this.brokers,
+      retry: {
+        initialRetryTime: 3000,
+        retries: 15
```

Comment:
checkout-api not getting ready during container start while Kafka broker is loading